packages/webpack-loader/test/typescript-loader.unit.ts

```diff
@@ -2,6 +2,7 @@ import { join } from 'path';
 import { expect } from 'chai';
 import { fixturesRoot } from '@ts-tools/fixtures';
 import { bundleWithLoader } from './bundle-with-loader';
+import ts from 'typescript';
```

Comment:
lib imports above local imports (and below node ones).
sorry, I should find/add a lint rule for that.